src/app/site/cart/components/cart-items/cart-items.component.ts

```diff
@@ -36,13 +36,8 @@ export class CartItemsComponent implements OnInit {
     this.displayTotalPrice = priceFormatter(this.product.quantity * this.product.price) + ' تومان';
 
     this.auth.isLoggedIn.subscribe(() => {
-      const isEU = this.auth.userDetails.shoesType === 'EU';
-      if (isEU) {
-        const gender = this.product.tags.find(tag => tag.tg_name.toUpperCase() === 'GENDER').name;
-        this.displaySize = this.dict.USToEU(this.product.size, gender);
-      } else {
-        this.displaySize = this.dict.translateWord(this.product.size);
-      }
+      const gender = this.product.tags.find(tag => tag.tg_name.toUpperCase() === 'GENDER').name;
```

Comment:
it's good to have a safeguard here in case `find` returns `undefined` - to avoid getting error of `Cannot read property 'name' of undefined`